### PR TITLE
Drop support for Kubernetes 1.9

### PR DESCRIPTION
**Pull request by rfranzke**
**What this PR does / why we need it**:
Drop support for Kubernetes 1.9
Usage was also not recommended due to CVE-2018-1002105.

**Release note**:

```action operator
:warning: Gardener does no longer support Kubernetes v1.9. You must upgrade all shoot cluster to at least Kubernetes v1.10 before upgrading Gardener. This also means that no seed clusters with versions prior to v1.10 are supported anymore. Please upgrade all seed clusters to at least v1.10 as well before upgrading Gardener.
```

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Gardener  
-Copyright (c) 2017-2018 SAP SE or an SAP affiliate company. All rights reserved.     
+Copyright (c) 2017-2019 SAP SE or an SAP affiliate company. All rights reserved.
```
Comment (adracus) [inline NOTICE.md:2]: :fireworks: New year, updated Gardener copyright :grin:

File: pkg/apis/garden/validation/validation_test.go [hunk 2]
```diff
@@ -4842,41 +4842,38 @@ var _ = Describe("validation", func() {
 
 				errorList := ValidateShoot(shoot)
 
-				Expect(len(errorList)).To(Equal(7))
-				Expect(*errorList[0]).To(MatchFields(IgnoreExtras, Fields{
+				fmt.Printf("%+v", errorList)
```
Comment (adracus) [inline pkg/apis/garden/validation/validation_test.go]: Leftover?

Comment (rfranzke) [reply pkg/apis/garden/validation/validation_test.go]: nice catch, removed

**Review by mvladev — APPROVED** [review]
/lgtm

**Review by adracus — APPROVED** [review]

**Pull request merged**